src/navigation/dispatch/components/GroupedTasks.tsx

```diff
@@ -55,6 +55,10 @@ export default function GroupedTasks({
   const navigation = useNavigation();
   const tasksEntities = useSelector(selectTasksEntities);
   const allTaskLists = useSelector(selectTaskLists);
+  const date = useSelector(selectSelectedDate)
+  const [generateOrders] = useRecurrenceRulesGenerateOrdersMutation()
+
+  useEffect(() => {generateOrders(date)}, [generateOrders, date]);
```

Comment:
Now you need to do here: `generateOrders(date.format('YYYY-MM-DD'))`
Do the change and plz check the call is being made fine.

Reply:
Done at https://github.com/coopcycle/coopcycle-app/pull/2027/commits/7a4ed39c847d5c32ab1380c9451325fc581724a4